docs/index.html

```diff
@@ -8,7 +8,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=Edge">
   <title>chota - A micro CSS framework</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./chota.css">
+  <link rel="stylesheet" href="https://unpkg.com/chota">
```

Comment:
oh man, I totally didn't catch this. Thanks for adding this but I'm going to do this change outside this PR and push it ASAP.